tests/common.sh

```diff
@@ -69,7 +69,7 @@ function get_pod () {
   pod_name="False"
   until [ $pod_name != "False" ] ; do
     sleep $sleep_time
-    pod_name=$(kubectl get pods -l $1 --namespace my-ripsaw -o name | cut -d/ -f2)
+    pod_name=$(kubectl get pods -l $1 --namespace ${3:-my-ripsaw} -o name | cut -d/ -f2)
```

Comment:
Don't quite follow the need for this change, or is this the result of not so perfect rebase

Reply:
This was my change. In the existing test code, the `get_pod` function is hard-coded to the `my-ripsaw` namespace. A test was added to the hammerdb CI script to check for deployment of the mssql server pod, which is in a different namespace. This function was extended to allow the namespace to be passed as a value, or otherwise default to `my-ripsaw`.